tests.Makefile

```diff
@@ -145,8 +145,8 @@ tests/py: $(PYTHON_TEST_TARGETS)
 $(PYTHON_TEST_TARGETS): %.__pytest__: .build/tests/py
 	$(info === Running tests in directory $* ===)
 	@docker run --rm \
-	  -v $(PWD):/data:ro \
-	  --entrypoint python3 \
+	  -v $(PWD):/data \
```

Comment:
can we keep read-only?

Reply:
The coverage tool needs write access in py3. It writes some intermediate metadata.